docs/Middlewares.mdx

```diff
@@ -131,7 +130,7 @@ authors: [hsluoyz]
 |---------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | [Actix](https://github.com/actix/actix)           | A Rust actors framework, via plugin: [actix-casbin](https://github.com/casbin-rs/actix-casbin)                                                                                                                                                                                                                                                                    |
 | [Actix web](https://github.com/actix/actix-web)   | A small, pragmatic, and extremely fast rust web framework, via plugin: [actix-casbin-auth](https://github.com/casbin-rs/actix-casbin-auth)                                                                                                                                                                                                                        |
-| [Rocket](https://github.com/SergioBenitez/Rocket) | a web framework for Rust that makes it simple to write fast, secure web applications without sacrificing flexibility, usability, or type safety, via plugin: [rocket-authz](https://github.com/casbin-rs/rocket-authz) or [rocket-casbin-auth](https://github.com/oiatz/rocket-casbin-auth) or [rocket-casbin-demo](https://github.com/simoin/Rocket_Casbin_Demo) |
```

Comment:
the author of `rocket-casbin-demo` seems to have deleted his account:  https://github.com/casbin/casbin-rs/issues/93#issuecomment-680218729

Reply:
Delete them